Web/Edubase.Web.UI/Controllers/Api/AcademyOpeningsApiController.cs

```diff
@@ -40,48 +40,61 @@ public class AcademyOpeningsApiController : ApiController
         /// <param name="to"></param>
         /// <param name="skip"></param>
         /// <param name="take"></param>
+        /// <param name="isSecure16To19User"></param>
+        /// <param name="establishmentTypeId"></param>
         /// <returns></returns>
-        [Route("api/academy-openings/list/{from:datetime}/{to:datetime}/{skip:int}/{take:int}"), HttpGet]
-        public async Task<dynamic> GetListAsync(DateTime from, DateTime to, int skip, int take)
+        [Route(
+             "api/academy-openings/list/{from:datetime}/{to:datetime}" +
+             "/{skip:int}/{take:int}/{isSecure16To19User}/{establishmentTypeId?}"),
```

Comment:
Route doesn't look right to me - I'll come back to it

Reply:
I realise now from going through my ticket with you, that this isn't a public-facing route, so the format probably doesn't matter